### ISSUE TITLE
[MESA][Linux] no 3d showing

### ISSUE THREAD

**Issue by DaanMeijer** [OpenSAGE/OpenSAGE#389]
When running in linux, with renderer set to OpenGL, there is nothing showing from the 3d renderer (only the menu overlay). Enabling OpenGL debugging yields the following output:
```
Mesa: User error: GL_INVALID_OPERATION in glGetFramebufferAttachmentParameteriv(invalid pname GL_FRAMEBUFFER_ATTACHMENT_DEPTH_SIZE)
Mesa: User error: GL_INVALID_OPERATION in glGetFramebufferAttachmentParameteriv(invalid pname GL_FRAMEBUFFER_ATTACHMENT_STENCIL_SIZE)
[22:46:55.8561][Warn] [CSF] Number of labels and strings mismatch.
[22:46:56.1091][Warn] Requested string 'MISSING: 'MAP:NoTest'' not found in 'en-US'.
[22:46:56.1091][Warn] Requested string 'MISSING: 'MAP:NoTest'' not found in 'en-US'.
[22:46:56.1091][Warn] Requested string 'MISSING: 'MAP:NoTest'' not found in 'en-US'.
[22:46:56.5652][Info] Weapon HumveeGun on game object  transitioning to state Inactive
Mesa: User error: GL_INVALID_VALUE in glCompressedTexSubImage2D(size=1024)
[22:46:56.5774][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.5830][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.5830][Info] Weapon HumveeGun on game object  transitioning to state Inactive
[22:46:56.5830][Info] Weapon HumveeGun on game object  transitioning to state Inactive
[22:46:56.5830][Info] Weapon HumveeGun on game object  transitioning to state Inactive
[22:46:56.6041][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.6045][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.6045][Info] Weapon HumveeGun on game object  transitioning to state Inactive
[22:46:56.6045][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.6082][Info] Weapon HumveeGun on game object  transitioning to state Inactive
[22:46:56.6082][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.6269][Info] Weapon HumveeGun on game object  transitioning to state Inactive
[22:46:56.6269][Info] Weapon CrusaderTankGun on game object  transitioning to state Inactive
[22:46:56.7109][Warn] Failed to resolve callback 'MainMenuUpdate'
[22:46:56.7109][Warn] Failed to resolve callback 'MainMenuShutdown'
[22:46:56.7177][Warn] Failed to resolve callback 'W3DShellMenuSchemeDraw'
[22:46:56.7177][Warn] Failed to resolve callback 'W3DClockDraw'
Mesa: 5 similar GL_INVALID_VALUE errors
Mesa: User error: GL_INVALID_OPERATION in active samplers with a different type refer to the same texture image unit
Mesa: User error: GL_INVALID_OPERATION in active samplers with a different type refer to the same texture image unit
Mesa: User error: GL_INVALID_OPERATION in active samplers with a different type refer to the same texture image unit
Mesa: User error: GL_INVALID_OPERATION in active samplers with a different type refer to the same texture image unit
Mesa: User error: GL_INVALID_OPERATION in active samplers with a different type refer to the same texture image unit
Mesa: User error: GL_INVALID_OPERATION in active samplers with a different type refer to the same texture image unit
```

**Comment by feliwir** [comment]
This only happens for MESA, so I'd restrict the issue to that

**Comment by feliwir** [comment]
Closed via #525
